UNL-CSE-AMBASSADORS/VITA
# 130 private queue error

Added in some error handling around the private queue methods.

**TESTING**
* Run DB refresher and build script
* Ensure all private queue functionality still works properly
  * Observe that after one has clicked a private queue button, you can no longer click it again
* Force an error on one of the queue methods (Just go put a `throw new Exception()` in the try-catch of any of the methods in `/server/queue/queue_priv.php`
  * Ensure an alert pops up with an error message and that the $scope.client.xxxx is reset to false and the button is enabled again 
* Ensure the public queue still functions
